internal/cucumber/config.go

```diff
@@ -142,6 +144,9 @@ func SetDefaults(p *Project) {
 				suite.PlatformName = "macOS 12"
 			}
 		}
+		if suite.PassThreshold < 1 {
+			suite.PassThreshold = 1
+		}
```

Comment:
Since we rely on the _retries_ setting now, we should also verify that the _pass threshold_ is never higher than _retries_.

Reply:
sure. let me add it